src/common/configure-packages.ts

```diff
@@ -29,7 +29,6 @@ export default function configurePackages() {
   // Docs: https://mobx.js.org/configuration.html
   Mobx.configure({
     enforceActions: "never",
-    isolateGlobalState: true,
```

Comment:
Seems like we don't need this: https://mobx.js.org/configuration.html#isolateglobalstate-boolean